latex2sympy.py

```diff
@@ -486,10 +486,10 @@ def convert_atom(atom):
             return sympy.Number(s)
     elif atom.DIFFERENTIAL():
         var = get_differential_var(atom.DIFFERENTIAL())
-        return sympy.Symbol('d' + var.name, real=True)
+        return sympy.Symbol('d' + var.name, real=True, positive=True)
     elif atom.mathit():
         text = rule2text(atom.mathit().mathit_text())
-        return sympy.Symbol(text, real=True)
+        return sympy.Symbol(text, real=True, positive=True)
     elif atom.VARIABLE():
```

Comment:
The one place that uses `Symbol` which I didn't mark as `positive` is for variable symbols

I didn't think it was needed, since we should be substituting values for them anyways, before we do any comparisons

Reply:
@builtbywill so just to make sure I'm following - did you just search for anywhere we use Symbol() and add positive=true? With this exception on variate symbols?

Reply:
@lily514 yep that's pretty much what I did 😂